lib/matplotlib/__init__.py

```diff
@@ -215,7 +217,78 @@ def _is_writable_dir(p):
     """
     return os.access(p, os.W_OK) and os.path.isdir(p)
 
+_verbose_msg = """--verbose-LEVEL and the Verbose class are deprecated.
+                  This functionality is now provided by the standard
+                  python logging library.  To get more (or less) logging
+                  output:
+                  `import logging`
+                  `logger=logging.getLogger('matplotlib')`
+                  `logger.set_level(logging.INFO)`
```

Comment:
import logging
logging.getLogger('matplotlib').setLevel(logging.INFO)

yes it's camelcase
and if you want to keep it over three lines, spaces around =